CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
 # Changelog
 
-=======
-
```

Comment:
<img width="254" alt="Screen Shot 2020-02-06 at 9 11 09 AM" src="https://user-images.githubusercontent.com/1738349/73949885-a527ef80-48c0-11ea-9031-4fa72204d06f.png">

For some reason some equal signs got in (likely from a rebase) causing this

Reply:
I'm guessing that was me!